### ISSUE TITLE
Spells Desync

### ISSUE THREAD

**Issue by Fuckenstain** [LightsHope/issues#722]
People Running with snares and stuns
Frostball has 60 yrds range warrior charge beats hunter multishot in range 
Charge Intercept Feral charge should not be slow when your charecter is slowed 
Feral charge snare effect can be resisted but not the silence effect

**Comment by Gamemechanicwow** [comment]
>Charge Intercept Feral charge should not be slow when your charecter is slowed

Yes, they should. It wasn't changed until patch 2.1: [Charge],  [Intercept] and  [Intervene] now suppress all Slowing effects while charging the target.

I know there is videos on youtube supporting, that slows affected charge during retail vanilla. I linked some last time a person claimed otherwise.
